### PR TITLE
fix: Replace toast with snackbar and format code

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Purpose / Description
Replace toast with snackbar.

## How Has This Been Tested?

Tested on Google emulator.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 1]
```diff
@@ -398,10 +399,9 @@ open class CardBrowser :
 
         override fun onSaveSearch(searchName: String?, searchTerms: String?) {
             if (searchName.isNullOrEmpty()) {
-                showThemedToast(
-                    this@CardBrowser,
-                    getString(R.string.card_browser_list_my_searches_new_search_error_empty_name),
-                    true
+                this@CardBrowser.showSnackbar(
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:402]: Is the `@CardBrowser` really needed?

**Review by lukstbit — COMMENTED** [review]
Is the @CardBrowser really needed?

**Comment by pratyaksh1610** [comment]
> Is the `@CardBrowser` really needed?

Yes, as a reference to the context.

**Comment by lukstbit** [comment]
> Yes, as a reference to the context.

Ok, but we don't need `this@CardBrowser` as the extension methods can be called directly, using `this@CardBrowser` it's more verbose and adds nothing. So please delete `this@CardBrowser` and call the `showSnackbar` methods directly.

**Review by lukstbit — APPROVED** [review]
Thank you for making the changes!

**Pull request merged**